src/main/java/net/neoforged/neoforge/event/AddPackFindersEvent.java

```diff
@@ -68,13 +71,22 @@ public void addPackFinders(ResourceLocation packLocation, PackType packType, Com
         if (getPackType() == packType) {
             var resourcePath = ModList.get().getModFileById(packLocation.getNamespace()).getFile().findResource(packLocation.getPath());
 
+            var version = ModList.get().getModContainerById(packLocation.getNamespace()).get().getModInfo().getVersion();
```

Comment:
You can avoid two lookups by storing the container and getting its mod info (and file)